cfme/automate/simulation.py

```diff
@@ -53,7 +56,11 @@ def simulate(**data):
     Args:
         **data: See :py:data:`sim_form` for keyword reference
     """
-    sel.force_navigate("automate_simulation")
+    try:
```

Comment:
i believe this is a place where `server = data.get('appliance', current_appliance).server` might be a compact and canonical way

@psav whats your POV on this?